crates/solver/src/driver/solver_settlements.rs

```diff
@@ -17,8 +17,10 @@ pub struct RatedSettlement {
     pub surplus: BigRational,     // In wei.
     pub earned_fees: BigRational, // In wei.
     pub solver_fees: BigRational, // In wei.
-    pub gas_estimate: U256,       // In gas units.
-    pub gas_price: BigRational,   // In wei per gas unit.
+    // solver estimated gas usage
```

Comment:
> we either use the protocol specified or solver specified (is passed in)

True, I was just looking into the future and wanted to avoid to have to remember to update the comment once the https://github.com/cowprotocol/services/issues/1821 is implemented.

Reply:
Just put a TODO(#1821) in the codebase and then when this PR is tackled it's easy to find all the pieces. More often than not, these things stay around longer than we would like.